pkg/flags/flagset.go

```diff
@@ -94,8 +94,8 @@ func ValidateFlags() error {
 		return fmt.Errorf("ValidateFlags called on unparsed flags")
 	}
 
-	if GitTrack == GTMEnabled && Namespace != "" && ClusterGitTrack == CGTMIncludeNamespaced {
-		return fmt.Errorf("Cannot use gittrack-mode enabled with namespace %q and clustergittrack-mode set to IncludeNamespaced", Namespace)
+	if GitTrack == GTMEnabled && Namespace != "" && ClusterGitTrack != CGTMDisabled {
```

Comment:
I think it's sufficient to just restrict to  namespace can't be used with ClusterGitTrack eg
```suggestion
	if Namespace != "" && ClusterGitTrack != CGTMDisabled {
```

I believe the problem would still occur even when `GitTrack != GTMEnabled` right?

Reply:
I think we technically allow `namespace` and `GTMDisabled` to be set at the same time, although I'm unsure what the behavior of that would be

Should we disallow that as well?

Reply:
I think disallow that, the namespace flag, as far as I know, is only useful for `GTMEnabled` and so should be disallowed otherwise I think